### PR TITLE
Improve the docs for Avo

### PR DESCRIPTION
**Pull request by bjornj12**
### Proposed changes

Updating the Docs for Avo, adding more intro and fixing the copy for some steps after feedback.

### Related issues (optional)

Based on updates from here:
https://github.com/segmentio/action-destinations/pull/1885

### CODE REVIEW

**Comment by bjornj12** [comment]
hey @forstisabella! Adding a little bit of improvements to the docs after review on the public page 🙌 


I have some questions though about the auto generated content


### Actions
How are the supported actions here detected? Currently we only support `Track` events, How would I change this to make sure it reflects that?
image


### Destination Settings
In our Destination, `App Version property` is clearly set to `required: false`, but somehow we're detected here as `Required` 
How do I get it changed to Optional?
image


### Engage
What is this engage? it is coming in because the docs say we support Identify? will it disappear if we do not support identify?
image




Thanks in advance! 🙏

**Comment by bjornj12** [comment]
bumping this @forstisabella, any chance to get a quick review on these things? we are hyping this this week 🙌

**Comment by forstisabella** [comment]
hi @bjornj12 - thanks for reaching out again about this! All of this information is pulled from Segment's Public API. When I look up the Avo destination using its ID (`65c2465d0d7d550aa8e7e5c6`), I get the following information about supported methods: 

```
"supportedMethods": {
                "track": true,
                "pageview": true,
                "identify": true,
                "group": true,
                "alias": true
            },
```

and the following for the appVersionPropertyName setting:

```
"name": "appVersionPropertyName",
                    "type": "string",
                    "defaultValue": "",
                    "description": "If you send a custom event property on all events that contains the app version, please enter the name of that property here (e.g. “app_version”). If you do not have a custom event property for the app version, please leave this field empty.",
                    "required": true,
                    "label": "App Version Property"
                },
```

I see that [the PR you opened in the actions-destinations repo](https://github.com/segmentio/action-destinations/pull/1885/files) was only recently merged - when we re-run our catalog as part of our regular docs release process, the appVersionPropertyName will programmatically be updated to be 'Optional". 

However, to update the supported methods, you'd need to reach out to the folks that run the partner portal. 

@tcgilbert - what would Bjorn need to do to update the supported methods? 

In terms of Engage support - that's inferred from platform support. Given that the destination supports the server platform, the site infers that it can support Engage. If that isn't the case, please let me know and I can open an additional PR to add the destination ID to an override list to remove this section. 

If you have any additional questions, please reach out!

**Comment by tcgilbert** [comment]
> How are the supported actions here detected? Currently we only support `Track` events, How would I change this to make sure it reflects that?

@bjornj12 I updated this to only show `Track`. This will be reflected when the docs page next rebuilds. If you need this updated in the future, please reach out to partner-support@gmail.com.

**Comment by bjornj12** [comment]
Great @forstisabella , @tcgilbert thanks!

re: appVersionPropertyName

This is the config field in the PR that was merged 🤔 Will it automatically detect in next build?
```
 appVersionPropertyName: {
        label: 'App Version Property',
        description:
          'If you send a custom event property on all events that contains the app version, please enter the name of that property here (e.g. “app_version”). If you do not have a custom event property for the app version, please leave this field empty.',
        type: 'string',
        required: false
      }
```
Re: Engage 

>  Given that the destination supports the server platform, the site infers that it can support Engage.

Not sure whether we support it or not. If it's inferred by server functions, I guess we do?


Is it possible to get this PR in a speedy review and ship and we iterate in different PR if the app Version is not changing to optional?
We are hyping these docs up in a blog post later today 🙌 pushing everyone to segment as the easiest way to install the Inspector for Avo 🚀

File: src/connections/destinations/catalog/avo/index.md [hunk 1]
```diff
@@ -5,20 +5,20 @@ beta: true
...
 
-
 {% include content/plan-grid.md name="actions" %}
 
 ## Supported methods
 
 ### Track events
 
-The Avo destination supports Track events.
+The Avo destination supports `Track` events.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/avo/index.md:21]: When discussing Segment methods outside of code snippets, please capitalize them and avoid stylizing them like inline code.

```suggestion
The Avo destination supports Track events.
```

File: src/connections/destinations/catalog/avo/index.md [hunk 2]
```diff
@@ -65,9 +65,27 @@ Before connecting the segment source to Avo, you will need an API key for your s
...
+You can copy the API key from your source in Avo. The API key allows Avo to map the events from your Segment source to the Avo source, to accurately compare your source’s event schemas to your Tracking Plan in Avo.
+
+#### Environment
+
+Environment describes which app environment the source is sent from, `Development | Staging | Production`.
+Avo only generates issues for events in the `Production` environment, but you can see the event shapes for staging and development environments to make sure they are implemented correctly.
+
+#### App Version Property
+
+App Version Property is an optional **(but recommended!)** field. Having accurate app release versions in Avo Inspector allows you to see how events change across releases. This will help you identify which releases an issue is impacting, and monitor for regressions in future releases after an issue has been resolved.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/avo/index.md:81]:
```suggestion
App Version Property is an optional **(but recommended!)** field. Having accurate app release versions in Avo Inspector allows you to see how events change across releases. This helps you identify which releases an issue is impacting, and monitor for regressions in future releases after an issue has been resolved.
```

File: src/connections/destinations/catalog/avo/index.md [hunk 3]
```diff
@@ -65,9 +65,27 @@ Before connecting the segment source to Avo, you will need an API key for your s
...
+### Configure Destination
+
+#### Avo Inspector API Key
+
+You can copy the API key from your source in Avo. The API key allows Avo to map the events from your Segment source to the Avo source, to accurately compare your source’s event schemas to your Tracking Plan in Avo.
+
+#### Environment
+
+Environment describes which app environment the source is sent from, `Development | Staging | Production`.
+Avo only generates issues for events in the `Production` environment, but you can see the event shapes for staging and development environments to make sure they are implemented correctly.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/avo/index.md:77]:
```suggestion
Avo only generates issues for events in your `Production` environment, but you can see the event shapes for staging and development environments to make sure they are implemented correctly.
```

File: src/connections/destinations/catalog/avo/index.md [hunk 4]
```diff
@@ -47,15 +47,15 @@ This Track call is translated into a event signature that is sent to Avo's Inspe
 
 1. From the Segment web app, click **Catalog**, then click **Destinations**.
 2. Find the Destinations Actions item in the left navigation, and click it.
-3. Select Avo from the list of destinations, then click **Add destination**.
-4. Select a source to connect to Avo (Actions) and click **Next**. 
-5. Enter a name for your Avo (Actions) destination and click **Create destination**. 
+3. Select [Avo](https://app.segment.com/goto-my-workspace/destinations/catalog/actions-avo) from the list of destinations, then click **Add destination**.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/avo/index.md:50]:
```suggestion
3. Select [Avo](https://app.segment.com/goto-my-workspace/destinations/catalog/actions-avo){:target="\_blank”} from the list of destinations, then click **Add destination**.
```

**Comment by forstisabella** [comment]
> Great @forstisabella , @tcgilbert thanks!
> 
> re: appVersionPropertyName
> 
> This is the config field in the PR that was merged 🤔 Will it automatically detect in next build?
> 
> ```
>  appVersionPropertyName: {
>         label: 'App Version Property',
>         description:
>           'If you send a custom event property on all events that contains the app version, please enter the name of that property here (e.g. “app_version”). If you do not have a custom event property for the app version, please leave this field empty.',
>         type: 'string',
>         required: false
>       }
> ```
> 
> Re: Engage
> 
> > Given that the destination supports the server platform, the site infers that it can support Engage.
> 
> Not sure whether we support it or not. If it's inferred by server functions, I guess we do?
> 
> Is it possible to get this PR in a speedy review and ship and we iterate in different PR if the app Version is not changing to optional? We are hyping these docs up in a blog post later today 🙌 pushing everyone to segment as the easiest way to install the Inspector for Avo 🚀

It should automatically update when the catalog updates, yes! 

I've completed a review - please take a look at my feedback so we can get this merged today!

**Comment by forstisabella** [comment]
Hi @bjornj12 - looks like your PR has branch protections, so I'm unable to edit it directly. I've opened another PR with the same information: #6092. I'm closing this PR for now!

**Comment by bjornj12** [comment]
You are a hero @forstisabella ! 🙌

**Pull request closed without merging**